sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeTableExec.scala

```diff
@@ -43,7 +47,9 @@ case class DescribeTableExec(
   private def addTableDetails(rows: ArrayBuffer[InternalRow]): Unit = {
     rows += emptyRow()
     rows += toCatalystRow("# Detailed Table Information", "", "")
-    rows += toCatalystRow("Name", table.name(), "")
+    rows += toCatalystRow("Catalog", resolvedTable.catalog.name(), "")
+    rows += toCatalystRow("Database", resolvedTable.identifier.namespace().mkString("."), "")
```

Comment:
is namespace() returning a list? Should this be a single identifier?

Reply:
Yes, namespace() returns a list. It's a feature of DS V2. Usually the length should be 1.

Reply:
so if it returns a list will the Database name contain periods?

Reply:
If the length of the namespace is 1, there is no any period

Reply:
we spoke online with Wenchen and Gengliang, didn't realize the namespace is a multi part name.